editor/editor_node.cpp

```diff
@@ -5262,13 +5262,15 @@ EditorNode::EditorNode() {
 	}
 
 	scene_tree_dock = memnew(SceneTreeDock(this, scene_root, editor_selection, editor_data));
-	scene_tree_dock->set_name(TTR("Scene"));
+	scene_tree_dock->set_name("Scene");
```

Comment:
That name should likely be set directly in `SceneTreeDock`'s constructor no? Same for others.

Reply:
Only the Node dock was set in the constructor before, but if you want them all to be, sure.